readthedocs/rtd_tests/tests/test_post_commit_hooks.py

```diff
@@ -8,6 +8,101 @@
 log = logging.getLogger(__name__)
 
 
+class GitLabWebHookTest(TestCase):
+    fixtures = ["eric", "test_data"]
+
+    def tearDown(self):
+        tasks.update_docs = self.old_bd
+
+    def setUp(self):
+        self.old_bd = tasks.update_docs
+
+        def mock(*args, **kwargs):
+            log.info("Mocking for great profit and speed.")
+        tasks.update_docs = mock
+        tasks.update_docs.delay = mock
+
+        self.client.login(username='eric', password='test')
+        self.payload = {
+            "object_kind": "push",
+            "before": "95790bf891e76fee5e1747ab589903a6a1f80f22",
+            "after": "da1560886d4f094c3e6c9ef40349f7d38b5d27d7",
+            "ref": "refs/heads/awesome",
+            "user_id": 4,
+            "user_name": "John Smith",
+            "user_email": "john@example.com",
+            "project_id": 15,
+            "repository": {
+                "name": "Diaspora",
+                "url": "git@github.com:rtfd/readthedocs.org.git",
```

Comment:
Is this a real POST request from a gitlab commit hook? I'm asking because it includes github.com URLs which seem a bit odd. It won't make any difference in the logic but it will be confusing for future developers who look at the tests and see github URLs in the tests for gitlab.